### DISCUSSION TITLE
Bookmark:  Acrobat7HighQualityPrint.joboptions

### DISCUSSION THREAD

**Discussion by ASM-SW** [sumatrapdfreader/sumatrapdf#2602] [Q&A]
When viewing a PDF with Sumatra I see the following bookmark at the end of the list:  
  Acrobat7HighQualityPrint.joboptions
I do not see this bookmark when I use Acrobat DC.  I attached an example screen snap showing the extra entry.  My mouse is hovering over the link and shows the help text.

![Bookmark](https://user-images.githubusercontent.com/23287756/170502657-a20c226a-2a43-46ca-b038-6ae862c4704d.png)

The entry does nothing.  When I hover over it it shows the name of the file

**Comment by GitHubRulesOK** [comment]
SumatraPDF shows internal attachments as bookmarks they may be invoice.xml, video.mp4 any.docx retired.SWF or in this case instructions for the printshop.

**Reply by ASM-SW** [reply]
In this case this bookmark is an annoyance.  Is there a way to prevent it from being shown in SumatraPDF?
Thanks

**Reply by kjk** [reply]
No, there is no way to prevent it being shown.
